api-reference/10 UI Components/dxStepper/9 Types/DisposingEvent/DisposingEvent.md

```diff
@@ -9,7 +9,7 @@ generateTypeLink:
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+The type of the [disposing]({basewidgetpath}/Events/#disposing) event handler's argument.
```

Comment:
This is a complicated possessive. And I see it repeats many times. I don't know if entire DevExtreme documentation maybe uses the same pattern. But I would change it to something like this: 

```suggestion
The argument type in the [disposing]({basewidgetpath}/Events/#disposing) event.
```

If that version works, please apply to other similar descriptions.